pkg/guestagent/kubernetesservice/kubernetesservice.go

```diff
@@ -131,11 +131,6 @@ func (s *ServiceWatcher) GetPorts() []Entry {
 		}
 
 		for _, portEntry := range service.Spec.Ports {
-			if portEntry.Protocol != corev1.ProtocolTCP {
```

Comment:
add a check if grpc vs ssh is being used?

Reply:
If the guest agent knows it would be nice to skip only when using the builtin port forwarder. But it is more important to get this working first, optimizing can be done later.